recipes-omnect/omnect-device-service/omnect-device-service.inc

```diff
@@ -26,20 +23,33 @@ LIC_FILES_CHKSUM = " \
 "
 
 DEPENDS += "\
-    azure-iot-sdk-c\
+    azure-iot-sdk-c \
+    pkgconfig-native \
 "
+
 RDEPENDS:${PN} += "\
     aziot-identityd\
     ca-certificates\
+    polkit \
+    sudo \
```

Comment:
just for my interest - why wasn't that needed before?

Reply:
regarding `polkit`: i moved that from the image recipe, because the only reason we use it, is `omnect-device-service`.
regarding `sudo`: its part of the image recipe and it's essential that it is (so i didn't move that). I added it here, because it is a runtime dependency.